notebook_analyzer/connector/connector.py

```diff
@@ -5,9 +5,9 @@
 
 class Connector:
 
-    def __init__(self, notebook_id, db_name):
+    def __init__(self, notebook_id, db_name=""):
         self.data = None
-        self.engine = create_engine(db_name)
+        self.engine = create_engine(db_name) if db_name else None
```

Comment:
Could you comment this change- dont catch the logic behind this

Reply:
This is because we want to be able to get aggregated metrics without saving them to the database.  it will probably be better to save them separately like cells